src/EditorFeatures/CSharpTest/Diagnostics/DiagnosticAnalyzerDriver/DiagnosticAnalyzerDriverTests.cs

```diff
@@ -656,21 +656,27 @@ await TestNuGetAndVsixAnalyzerCoreAsync(
         //   1) No duplicate diagnostics
         //   2) Both NuGet and Vsix analyzers execute
         //   3) Appropriate diagnostic filtering is done - Nuget suppressor suppresses VSIX analyzer.
+        //
+        // 🐛 After splitting fallback options into separate CompilationWithAnalyzers for project and host analyzers,
```

Comment:
Link issue

Reply:
I will create an issue to link if I don't find a clean way to fix this before merging.

Reply:
➡️ Link added to #75399